### DISCUSSION TITLE
Typo in README

### DISCUSSION THREAD

**Discussion by jrygeorge** [marimo-team/marimo#1740] [General]
Under the _"Compatible with expensive notebooks."_ heading, shouldn't it say "...and mark affected <ins>cells</ins> as stale..." instead of  "...and mark affected <ins>stales</ins> as stale.." ?

**Comment by akshayka** [comment]
Oops! Thank you for letting us know; fixed. Next time you can open questions like this as a GitHub issue -- more likely to see it there.
